pollux/lib/core/src/main/scala/io/iohk/atala/pollux/core/service/PresentationServiceImpl.scala

```diff
@@ -288,7 +322,7 @@ private class PresentationServiceImpl(
                 `@context` = Vector("https://www.w3.org/2018/presentations/v1"),
                 maybeId = None,
                 `type` = Vector("VerifiablePresentation"),
-                verifiableCredential = vcs.toVector,
+                verifiableCredential = ???, // FIXME vcs.toVector,
                 holder = prover.did.value,
                 verifier = Vector("https://example.verifier"), // TODO Fix this
```

Comment:
What is the point of this?
`verifier = Vector(options.domain),`
If the options are missing should this become an error?